src/nvim/highlight_group.c

```diff
@@ -226,12 +229,16 @@ static const char *highlight_init_both[] = {
   "default link @keyword Keyword",
   "default link @exception Exception",
 
+  "default link @variable Identifier",
```

Comment:
This causes the typical "everything has one color" problem. Identifier has been traditionally used for something to stand out while `@variable` is almost used for everything.

`@tag` is used in nvim-treesitter exclusively for hmtl-tags

Reply:
These are just fallbacks. The expectation is that virtually everyone that uses treesitter highlighting will define the highlight groups to taste; either themselves or via a colorscheme.

Reply:
> @tag is used in nvim-treesitter exclusively for hmtl-tags

And that is one reason why I pushed for rethinking the captures from scratch; these "single purpose" captures go against your stated goal of uniformity.

(Helix uses a `@markup` top-level capture for such things, but this discussion is off-topic here and should be had in chat first.)